README.md

```diff
@@ -245,8 +245,55 @@ An XML variant may look like:
 ```
 
 Note: by default the message `payload` is a `Map` converted from the received Topic JSON message. For the convenient
-the `payload-expression` is provided with the `Message` as a root object of the evaluation context. Hence even some 
-HTTP headers, populated by the `DefaultHttpHeaderMapper`, are available for the evaluation context.  
+the `payload-expression` is provided with the `Message` as a root object of the evaluation context. Hence even some
+HTTP headers, populated by the `DefaultHttpHeaderMapper`, are available for the evaluation context.
+
+###Outbound Channel Adapter
+
+The `SnsMessageHandler` (`<int-aws:sns-outbound-channel-adapter>`) is pretty simple one-way Outbound Channel Adapter
+to send Topic Notification using `AmasonSNS` service.
+
+This Channel Adapter (`MessageHandler`) accepts these options:
+
+- `topic-arn` (`topic-arn-expression`) - the SNS Topic to send notification for. The `ResourceIdResolver` can be used
+from the SpEL definition to determine the target Topic Arn from the local logical name;
+- `subject` (`subject-expression`) - the SNS Notification Subject;
+- `body-expression` - the SpEL expression to evaluate the `message` property for the
+`com.amazonaws.services.sns.model.PublishRequest`.
+
+See `SnsMessageHandler` JavaDocs for more information.
+
+The Java Config looks like:
+
+````java
+@Bean
+public MessageHandler snsMessageHandler() {
+   	SnsMessageHandler handler = new SnsMessageHandler(amazonSns());
+   	adapter.setTopicArn("arn:aws:sns:eu-west:123456789012:test);
+   	String bodyExpression = "SnsBodyBuilder.withDefault(payload).forProtocols(payload.substring(0, 140), 'sms')";
+    handler.setBodyExpression(spelExpressionParser.parseExpression(bodyExpression));
+   	return handler;
+}
+````
+
+NOTE: the `bodyExpression` can be evaluated to the `org.springframework.integration.aws.support.SnsBodyBuilder`
+allowing to configure `json` `messageStructure` for the `PublishRequest` and provide separate messages for different
+protocols. The same `SnsBodyBuilder` rule is applied for the raw `payload` if the `bodyExpression` hasn't been
+configured.
+NOTE: if the `payload` of `requestMessage` is a `com.amazonaws.services.sns.model.PublishRequest` already,
+the `SnsMessageHandler` doesn't do anything with it and send publishes it to SNS as is.
```

Comment:
s/can be evaluated to the/can be evaluated to a/
s/allowing to configure/allowing the configuration of a/
s/send publishes it to SNS as is/it is sent as-is/